requirements/base.in

```diff
@@ -17,3 +17,6 @@
 # under the License.
 #
 -e file:.
+paramiko>=3.4.0
```

Comment:
Why aren't these in `setup.py`? It seems like we use them (well at least some of them) directly:

```
$ git grep paramiko
superset/extensions/ssh.py:from paramiko import RSAKey
```

Reply:
no idea.
Added it